Floofbot/Modules/Conversions.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Discord;
+using Discord.Addons.Interactive;
+using Discord.Commands;
+using UnitsNet;
+
+namespace Floofbot
+{
+    [Summary("Conversion commands")]
+    [Discord.Commands.Name("Conversions")]
+    public class Conversions : InteractiveBase
+    {
+        private static readonly Discord.Color EMBED_COLOR = Color.Magenta;
+
+        [Command("convert")]
+        [Alias("conv")]
+        [Summary("Converts units to other units, such as Celcius to Fahrenheit.")]
+
+        public async Task convert(string input)
+        {
+            string fahPat = "\\b(\\d+)(f)\\b";
+            Regex fahReg = new Regex(fahPat, RegexOptions.IgnoreCase);
+            string celPat = "\\b(\\d+)(c)\\b";
+            Regex celReg = new Regex(celPat, RegexOptions.IgnoreCase);
+            string miPat = "\\b(\\d+)(mi)\\b";
+            Regex miReg = new Regex(miPat, RegexOptions.IgnoreCase);
+            string kmPat = "\\b(\\d+)(km)\\b";
+            Regex kmReg = new Regex(kmPat, RegexOptions.IgnoreCase);
+            string kgPat = "\\b(\\d+)(kg)\\b";
+            Regex kgReg = new Regex(kgPat, RegexOptions.IgnoreCase);
+            string lbPat = "\\b(\\d+)(lb)||(lbs)\\b";
+            Regex lbReg = new Regex(lbPat, RegexOptions.IgnoreCase);
+
+            if (fahReg.Match(input).Success)
```

Comment:
Might be worth adding an else statement. One that lists the available conversions.

Reply:
I just fixed that, along with compressing the code.